src/tests/test_cases/test.py

```diff
@@ -53,6 +54,21 @@ def nested():
         ''
 
 
+@overload
+def overloaded_func(a: int) -> str:
+    ...
+
+
+@overload
+def overloaded_func(a: str) -> str:
```

Comment:
What happens here if you do add a docstring?

Reply:
You would get the expected D418 error.

Reply:
How about that, I was sure that the @expect decorator is the only way to expect in this module, I was wrong...